v2/tools/generator/internal/armconversion/convert_to_arm_function_builder.go

```diff
@@ -371,7 +371,7 @@ func (builder *convertToARMBuilder) convertReferenceProperty(_ *astmodel.Convers
 	}
 
 	// Don't need to worry about conflicting names here since the property name was unique to begin with
-	localVarName := builder.idFactory.CreateIdentifier(params.NameHint+"ARMID", astmodel.NotExported)
+	localVarName := builder.idFactory.CreateLocal(params.NameHint + "ARMID")
```

Comment:
This isn't gofmt'ed, right? At least it doesn't matching the formatting of the line before modification?

Please don't tell me the rules change if there's only 1 parameter? Uggh.

Reply:
Ok, I won't tell you the rules change if there's only one parameter.